packages/dev/core/src/Shaders/ShadersInclude/pbrBlockReflectivity.fx

```diff
@@ -117,17 +119,21 @@ reflectivityOutParams reflectivityBlock(
             // Compute the converted reflectivity.
             surfaceReflectivityColor = mix(0.16 * reflectance * reflectance, baseColor, metallicRoughness.r);
         #else
-            vec3 metallicF0 = metallicReflectanceFactors.rgb;
-
             #if DEBUGMODE > 0
-                outParams.metallicF0 = metallicF0;
+                outParams.metallicF0 = vec3(metallicReflectanceFactors.a);
             #endif
 
             // Compute the converted diffuse.
-            outParams.surfaceAlbedo = mix(baseColor.rgb * (1.0 - metallicF0), vec3(0., 0., 0.), metallicRoughness.r);

```

Comment:
This `(1.0 - metallicF0)` factor isn't correct. It compensates for energy lost to specular at F0 but ignores energy lost at other angles.